Add GW status to feedback info

pablothedude opened the pull request:
The response from GW needed to be shown as additional info on
the error pages.

https://www.pivotaltracker.com/story/show/167834563

MKodde commented on src/OpenConext/EngineBlockBundle/Controller/FeedbackController.php:
```diff
@@ -453,8 +453,10 @@ public function clockIssueAction()
      */
     public function stepupCalloutUserCancelledAction(Request $request)
     {
+        $feedbackInfo = $request->getSession()->get('feedbackInfo');
```
No need to change the feedback controller. The Feedback info is not passed as a twig param, but fished out of the session by the Feedback twig extension.

pablothedude replied:
A nice that we have a TD ticket already then!

thijskh commented:
Re 2: you could consider to always strip off any `urn:oasis:names:tc:SAML:2.0:status:` prefix of the statuscode and substatus code before displaying. I would be fine with that.

pablothedude commented on src/OpenConext/EngineBlockBundle/Controller/StepupController.php:
```diff
@@ -65,6 +67,21 @@ public function __construct(
     {
+
+
+        $statusCodeDescription = 'Responder/AuthnFailed';
```
Removed

pablothedude commented on library/EngineBlock/Corto/Module/Service/StepupAssertionConsumer.php:
```diff
@@ -167,13 +168,15 @@ private function handleInvalidGatewayResponse(EngineBlock_Corto_Exception_Receiv
                 }
 
                 throw new EngineBlock_Corto_Exception_InvalidStepupLoaLevel(
+                    $e,
```
Fixed

MKodde commented on src/OpenConext/EngineBlockBundle/Controller/StepupController.php:
```diff
@@ -21,6 +21,8 @@
 
 use EngineBlock_ApplicationSingleton;
 use EngineBlock_Corto_Adapter;
+use EngineBlock_Corto_Exception_ReceivedErrorStatusCode;
```
These use statements are now unused and should be dropped from this commit

pablothedude replied:
done

pablothedude commented:
> Almost there! See some code related feedback below.
> 
> In addition to that, some additional remarks:
> 
>     1. The status code and status message feedback information labels might need an abbreviation.
> 
The fields were already implemented without an abbreviation. I have consulted Thijs and for now it was fine to keep it as-is to prevent becoming to cryptic. This could easiliy be changed later on.

>     2. The status code field should be wider. The value does not fit in the cell.
> 
Done, seems to be forgotten when implemented the first time. Uses the full width now.

>     3. I miss the addition of the stepup link as per requirement: `Provide link to page (https://support.surfconext.nl/stepup-noauthncontext) that can be customized with advice e.g. the right link to the self-service portal and add translation.`
> 
Added the link trough translations

>     4. Consider updating the behat test, covering the error pages with a verification that the status code & message is rendered on the stepup pages.
Added the status code to the tests.
